src/app/styles/constants.ts

```diff
@@ -0,0 +1 @@
+export const COLOR_NEGATIVE_AMOUNT = 'red';
```

Comment:
why is this red and not the color coming from the theme?
which is a different kind of red
https://projects.invisionapp.com/d/main#/console/18925362/394271989/preview